hvac/v1/__init__.py

```diff
@@ -764,8 +765,18 @@ def create_ec2_role(self, role, bound_ami_id=None, bound_account_id=None, bound_
             params['bound_iam_instance_profile_arn'] = bound_iam_instance_profile_arn
         if role_tag is not None:
             params['role_tag'] = role_tag
+        if ttl is not None:
```

Comment:
From what I've seen, Vault is going to treat a None (null once its dumped into JSON) value the same as its default value of 0. How would you feel about dropping all this "if <param> is not None" boilerplate and instead just including them in the initial params dictionary declaration?